code/modules/awaymissions/corpse.dm

```diff
@@ -52,6 +52,7 @@
 	M.r_skin = rand (0,255) //skin coloring
 	M.g_skin = rand (0,255)
 	M.b_skin = rand (0,255)
+	M.s_tone = rand (-185,220) //skin tone

```

Comment:
Oops. That snuck in when I was trying to figure out if rand was a range or a starting point + random value.